lib/adb.js

```diff
@@ -50,6 +50,19 @@ class ADB {
     }
     this.executable.defaultArgs.push('-P', this.adbPort);
   }
+
+  /**
+   * Create a new instance of `ADB` that inherits configuration from this `ADB` instance.
+   * This avoids the need to call `ADB.createADB()` multiple times.
+   * @param {object} opts - Additional options mapping to pass to the `ADB` constructor.
+   * @returns {ADB} The resulting class instance.
+   */
+  clone (opts) {
+    const adb = new ADB(opts);
```

Comment:
Shouldn't we just copy opts from the existing instance instead of providing new ones?
It won't be an exact clone then